libs/dh/core/shell/src/lib/dh-core-shell.component.spec.ts

```diff
@@ -37,6 +40,9 @@ describe(DhCoreShellComponent.name, () => {
       By.directive(WattShellComponent)
     );
 
+    expect(
+      screen.queryByRole('heading', { name: /datahub \[da\]/i })
```

Comment:
question: can we override the default language to English in the Transloco testing module so that queries are for English translation texts?